style.css

```diff
@@ -781,3 +781,153 @@ p {
 .input1 > label {
   display: none;
 }
+
+/* Mobile Popup */
+
+.popUp {
+  position: fixed;
+  top: 0;
+  background-color: white;
+  padding: 24px;
+}
+
+.mobile-pop-up {
+  padding: 16px 16px 20px 16px;
+  border: 2px solid black;
+  margin: 60px 16px 211px 16px;
+  background-color: white;
+  max-width: 1156px;
+}
+
+.pop-up-top {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+  margin-right: 16px;
+  height: 82px;
+}
+
+.pop-up-h1 {
+  width: 80%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 20;
+  font-size: 25px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0;
+  margin-left: 30px;
+  padding-top: 10px;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` and `font size` etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-21 17-05-44](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/6633013c-ded1-46b9-afb3-35f8f5668a49)

</td>
<td>

![Screenshot from 2023-06-21 17-03-48](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/9e80755e-17b6-42f3-a6be-006da4352ddb)

</td>
</tr>
</table>